app/developers/templates/developers/add_question_condition_select_condition_type.html

```diff
@@ -0,0 +1,41 @@
+{% extends "deliver_grant_funding/grant_base.html" %}
+{% from "govuk_frontend_jinja/components/back-link/macro.html" import govukBackLink %}
+{% from "govuk_frontend_jinja/components/radios/macro.html" import govukRadios %}
+{% from "govuk_frontend_jinja/components/input/macro.html" import govukInput %}
+{% from "govuk_frontend_jinja/components/button/macro.html" import govukButton %}
+{% from "govuk_frontend_jinja/components/checkboxes/macro.html" import govukCheckboxes %}
+{% from "govuk_frontend_jinja/components/summary-list/macro.html" import govukSummaryList %}
+{% from "developers/_render_managed_expression_form.html" import render_managed_expression_form %}
+
+{% block pageTitle %}
+  Add a condition - {{ question.name }} - MHCLG Funding Service
+{% endblock pageTitle %}
+
+{% set active_item_identifier = "developers" %}
+
+{% block beforeContent %}
+  {# todo: pressing back here should remember which question you had selected in case you want to just move one up or down in the list #}
+  {#       that would also line up with how the browsers "back" button would work (remembering form options) #}
+  {{
+    govukBackLink({
+        "text": "Back",
+        "href": url_for("developers.add_question_condition_select_question", grant_id = grant.id, question_id=question.id)
+    })
+  }}
+{% endblock beforeContent %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">Add a condition</h1>
+
+      {% include "developers/_add_question_condition_context.html" %}
+
+      {% if form is not none %}
+        {{ render_managed_expression_form(form, label_html="Only show the question if the answer <span class='govuk-visually-hidden'>to “" ~ depends_on_question.text ~ "”</span> is") }}
```

Comment:
🤏 preference for HTML to not be inlined and to be set in a separate block so that text editors/ IDEs treat them appropriately 
```suggestion
        {% set label_html %}
          <span>Only show the question if the answer <span class='govuk-visually-hidden'>to “{{ depends_on_question.text }}”</span> is</span>
        {% endset %}
        {{ render_managed_expression_form(form, label_html=label_html) }}
```